src/fides/api/api/v1/endpoints/privacy_preference_endpoints.py

```diff
@@ -700,29 +704,39 @@ def validate_served_record(
             )
 
 
-def update_request_with_decoded_tc_string_fields(
+def update_request_with_decoded_fides_string_fields(
     request_body: PrivacyPreferencesRequest, db: Session
 ) -> PrivacyPreferencesRequest:
-    """Update the request body with the decoded values of the TC string if applicable"""
+    """Update the request body with the decoded values of the TC string and AC strings if applicable"""
     if request_body.fides_string:
         tcf_contents: TCFExperienceContents = get_tcf_contents(
             db
         )  # TODO cache this so we're not building each time privacy preference is saved
         try:
-            decoded_preference_request_body: TCStringFidesPreferences = (
-                decode_tc_string_to_preferences(request_body.fides_string, tcf_contents)
+            tc_str, ac_str = split_fides_string(request_body.fides_string)
+
+            decoded_tc_str_request_body: FidesStringFidesPreferences = (
+                decode_tc_string_to_preferences(tc_str, tcf_contents)
+            )
+            decoded_ac_str_request_body: FidesStringFidesPreferences = (
+                decode_ac_string_to_preferences(ac_str, tcf_contents)
+            )
+            # We combine Vendor Consent Preferences from the TC String and AC String if applicable
+            decoded_tc_str_request_body.vendor_consent_preferences = (
+                decoded_tc_str_request_body.vendor_consent_preferences
+                + decoded_ac_str_request_body.vendor_consent_preferences
```

Comment:
When saving preferences, I assume a fides_string can be passed in and split the string accordingly. It's backwards compatible. You can pass in just a TC string.  A comma-combined TC String and AC string. Or just an AC string even, provided a comma comes first.  It will fail early however if either the TC string or AC string are in an unexpected format.

The TC string is decoded per usual.  The AC string translation is new here, and I assume all pieces of the AC string will be saved as "vendor consents" specifically.  I combine the vendor consent preferences from the AC string and TC string into one.

The I proceed per usual, saving preferences, just like I would if the user had supplied them directly, rather than via the string.